packages/camera/src/__mocks__/i18n.js

```diff
@@ -0,0 +1,8 @@
+module.exports = () => ({
+  useTranslation: () => ({
+    t: (str) => str,
+    i18n: {
+      changeLanguage: () => new Promise(() => { }),
```

Comment:
When you mock a promise, instead of doing :

```javascript
() => new Promise(() => {});
```

Do something like :

```javascript
() => Promise.resolve();
```

Or :

```javascript
() => Promise.reject();
```